### PR TITLE
Quick Start v2: Add new fancy alerts

### PR DESCRIPTION
**Pull request by nheagy**
Refs #10860

This updates the existing quick start alert to match mockups, and adds the new mockup to alert users that quick start has been updated:

Updated Alert | New Alert
---- | ----
simulator screen shot - iphone xr - 2019-02-21 at 10 35 15 | simulator screen shot - iphone xr - 2019-02-21 at 10 55 11

### To test:
- Create a site with v1 Quick Start†
- Launch this branch
- Ensure that you see the new upgrade alert
- Create a new site with this branch
- Ensure that you see the new alert asking for v2 of quick start

### †How to create a v1 site:
- You could turn off the `.quickStartV2` flag, then:
  - create a new site
  - **or** drop a `QuickStartTourGuide.find()?.setup(for: blog)` somewhere you can trigger easily

Update release notes:

- [ ] If there are user facing changes, I have added an item to `RELEASE-NOTES.txt`.

### CODE REVIEW

File: WordPress/Classes/ViewRelated/System/FancyAlertViewController+QuickStart.swift [hunk 1]
```diff
@@ -1,5 +1,19 @@
 extension FancyAlertViewController {
     private struct Strings {
+        static let titleText = NSLocalizedString("Want a little help getting started?", comment: "Title of alert asking if users want to try out the quick start checklist.")
+        static let bodyText = NSLocalizedString("We’ll walk you through the basics of building and growing your site", comment: "Body text of alert asking if users want to try out the quick start checklist.")
+        static let allowButtonText = NSLocalizedString("Yes, help me", comment: "Allow button title shown in alert asking if users want to try out the quick start checklist.")
```
Comment (frosty) [inline WordPress/Classes/ViewRelated/System/FancyAlertViewController+QuickStart.swift]: The capitalization of the allow and not not buttons should probably be the same? (`Yes, help me` and `Not This Time`)

File: WordPress/Classes/ViewRelated/System/FancyAlertViewController+QuickStart.swift [hunk 2]
```diff
@@ -37,7 +54,67 @@ extension FancyAlertViewController {
...
+    ///
+    /// - Returns: FancyAlertViewController of the notice
+    @objc static func makeQuickStartUpgradeToV2AlertController(blog: Blog) -> FancyAlertViewController {
+        let okButton = ButtonConfig(UpgradeStrings.okButtonText) { controller, _ in
+            controller.dismiss(animated: true)
+        }
+
+        let image = UIImage(named: "wp-illustration-big-checkmark")
+
+        let config = FancyAlertViewController.Config(titleText: Strings.titleText,
```
Comment (frosty) [inline WordPress/Classes/ViewRelated/System/FancyAlertViewController+QuickStart.swift]: Should this be `UpdateStrings`?

**Comment by nheagy** [comment]
@frosty nice catch on the test scenario failing—I had removed a check for `.quickStartV2` 😬 I also made the other changes you suggested 😀 

This should be ready for re-review!

**Comment by nheagy** [comment]
Thanks @frosty!

**Pull request merged**